pkg/client/client.go

```diff
@@ -416,6 +416,15 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		fmt.Fprintf(w, "Masquerading:\t%s\n", status)
 	}
 
+	if sr.ClockSource != nil {
+		status := sr.ClockSource.Mode
+		if sr.ClockSource.Mode == models.ClockSourceModeJiffies {
+			status = fmt.Sprintf("%s\t[%d Hz]",
+				sr.ClockSource.Mode, sr.ClockSource.Hertz)
+		}
+		fmt.Fprintf(w, "Clock Source for BPF:\t%s\n", status)
```

Comment:
Should this be under `--verbose`? It looks minor compared to the other information we dump in `cilium status`. It's a single line so I don't have a strong opinion :slightly_smiling_face:

Reply:
No strong opinion. :) I can do that.